src/main/java/ca/ciccc/Assignment2.java

```diff
@@ -9,90 +10,160 @@
      */
     public boolean countNumbers(int[] nums) {
         // Todo: your code goes here
+        boolean onesGreater = false;
 
-        return false;
+        int oneNumber = 0;
+        int sevenNumber = 0;
+
+        for (int i = 0; i <= nums.length - 1; i++) {
+            if (nums[i] == 1) {
+                oneNumber += 1;
+            }
+            if (nums[i] == 7) {
+                sevenNumber += 1;
+            }
+        }
+        if (sevenNumber < oneNumber) {
+            onesGreater = true;
+        }
+        return onesGreater;
     }
 
     /**
      * Return the sum of numbers in the array, returning 0 for an empty array. Except the number 13
      * is very unlucky, so it does not count the number that comes immediately after a 13, also 13 itself
      * does not count.
+     *
      * @param nums array of numbers
      * @return the sum except for 13
-     *
+     * <p>
      * Sample Input / Output
      * sumExcept13({1, 2, 2, 1}) -> 6
      * sumExcept13({1, 2, 2, 1, 13} -> 6
      * sumExcept13({1, 2, 13, 2, 1, 13} -> 4
      */
     public int sumExcept13(int[] nums) {
         // Todo: your code goes here
-
-        return 0;
+        int sum = 0;
+        for (int i = 0; i < nums.length; i++) {
+            if (nums[i] == 13) {
+                sum = sum + 0;
+                i += 1;
+            } else {
+                sum = sum + nums[i];
+            }
+        }
+        return sum;
     }
 
     /**
      * Return an array that is "left shifted" by one -- so {6, 2, 5, 3} return {2, 5, 3, 6}.
      * You may modify and return the given array, or return a new array.
+     *
      * @param nums array of ints
      * @return shifted array
-     *
+     * <p>
      * Sample Input / Output
-     * shiftArray({6, 2, 5, 3}) -> {2, 4, 3, 6}
+     * shiftArray({6, 2, 5, 3}) -> {2, 5, 3, 6}
      * shiftArray({2, 3}) -> {3, 2}
      * shiftArray({7}) -> {7}
      */
-    public int[] shiftArray(int[] nums) {
+    public static int[] shiftArray(int[] nums) {
         // Todo: your code goes here
-
-        return null;
+        int[] shiftedArray = new int[nums.length];
+        if (nums.length == 0) {
+            return shiftedArray;
+        }
+        for (int i = 0; i < nums.length - 1; i++) {
+            shiftedArray[i] = nums[i + 1];
+        }
+        shiftedArray[shiftedArray.length - 1] = nums[0];
+        return shiftedArray;
     }
 
     /**
      * Return true if the array contains three increasing adjacent numbers, otherwise false.
+     *
      * @param nums array of ints.
      * @return true if the array contains three increasing adjacent numbers, otherwise false.
      */
     public boolean tripleIncreasingOrder(int[] nums) {
         // Todo: your code goes here
-
-        return false;
+        int count = 0;
+        for (int i = 0; i <= nums.length - 3; i++) {
+            if (nums[i] == (nums[i + 1] - 1) && nums[i] == (nums[i + 2] - 2)) {
+                count += 1;
+            } else {
```

Comment:
In this case, if array contains adjacent number more than three like {2,3,4,5,6,7}, it's gonna be false. I think you can return true when you find three increasing adjacent number.

Reply:
Thanks for your review!
I thought that we have to find just only three increasing numbers.
I'll change this code to return true at the case more than three numbers as well.